shell/plugins/dashboard-store/mutations.js

```diff
@@ -270,7 +270,11 @@ export default {
     const keyField = getters.keyFieldForType(type);
 
     allLatest.forEach((entry) => {
-      const existing = state.types[type].map.get(entry[keyField]);
+      let existing;
+
+      if (state.types && state.types[type]) {
+        existing = state.types[type].map.get(entry[keyField]);
+      }
```

Comment:
Had to implement this again since I got an error for a type that wasn't even defined. (don't remember which to be honest but it was early in the development)

There's a secondary effect that if `state.types[type]` throws an `undefined` console error, then the `load()` function doesn't even get exectuted which, imo, is far worst, as the `load()` seems to be capable of handling well a non-existing case @richard-cox

Reply:
I can try and replicate this again be reverting the changes 🤷

Reply:
reverting the code I was able to reproduce this for daemonsets. I think you are right, the problem is with the flow of the incremental loading (`loadDataPage`) where we simply don't register the type if it doesn't exist.
I remember that when I was developing the incremental + manual I had hit this but I could never found the root cause.

Maybe this is it. I've added it to the `loadDataPage` and we don't have any issue on the mutation.